magenta/models/melody_rnn/melody_rnn_generate.py

```diff
@@ -194,9 +198,11 @@ def run_with_flags(generator):
     # Set the start time to begin on the next step after the last note ends.
     last_end_time = (max(n.end_time for n in primer_sequence.notes)
                      if primer_sequence.notes else 0)
+    start_time = last_end_time + _steps_to_seconds(1, qpm)
+
     generate_section = generator_options.generate_sections.add(
-        start_time=last_end_time + _steps_to_seconds(1, qpm),
-        end_time=total_seconds)
+      start_time=start_time,
```

Comment:
we use 4 spaces for line continuations, so you'll need to add another 2 spaces before these 2 lines.